importer/src/cli.ts

```diff
@@ -0,0 +1,21 @@
+import { program } from 'commander';
+import { run } from './cli-run.js';
+import { IMPORTER_VERSION } from './version.js';
+
+program
+  .name('xata-importer')
+  .description('Importer utility for xata.io')
+  .version(IMPORTER_VERSION)
+  .argument('<file>', 'File containing the data to import. Pass "-" to pipe content instead.')
+  .option('--table <table>', 'The table in which to import the data.')
+  .option('--columns <columns>', 'List of column names.')
+  .option('--types <types>', 'List of column names.')
+  .option('--noheader', 'Specify that the CSV file has no header.')
```

Comment:
I would have expected with this flag to use the existing table columns, but I guess order cannot be guaranteed. Could we make it interactive somehow? [just a comment, but I understand this as a future improvement]